assets/collection-page.css

```diff
@@ -63,12 +84,44 @@
 
   .collection-page__grid .collection-page__products {
     grid-area: grid;
-    grid-auto-rows: fit-content(100%);
   }
 
   .collection-page__grid .collection-page__pagination {
     grid-area: pagination;
   }
+
+  .collection-page__wrapper > .collection-page__products,
+  .collection-page__wrapper > .collection-page__grid {
+    padding: 0 24px;
+  }
+
+  .collection-page__wrapper .product-card__vision:after {
+    aspect-ratio: 1 / 0.975;
+  }
+
+  .collection-page__container .product-card__vision:after {
+    aspect-ratio: 1 / 1.036;
+  }
+
+  .collection-page__wrapper-three .product-card__vision:after {
+    aspect-ratio: 1 / 1.407;
+  }
+
+  .collection-page__wrapper-three .collection-page__container .product-card__vision:after {
+    aspect-ratio: 1 / 1.2825;
+  }
+
+  .collection-page__wrapper-four .product-card__vision:after {
+    aspect-ratio: 1 / 1.275;
+  }
+
+  .collection-page__wrapper-four .collection-page__container .product-card__vision:after {
+    aspect-ratio: 1 / 1.32155;
+  }
+
+  .collection-page__wrapper-four .collection-page__container .collection-page__top ~ .collection-page__grid .product-card__vision:after {
+    aspect-ratio: 1 / 1.55;
+  }
```

Comment:
Are these aspect ratios defined by design? Looking a bit weird

Reply:
There are 2 designs with a layout of 2 and 4 products in a row.
I'd say I had to add a few more CSS scenarios because the product pictures look too small especially when the sidebar with menu is exist on page